rear_camera_py/Makefile

```
@@ -7,8 +11,18 @@ help:
 	@echo " $$ make rm"
 	@echo "フォーマットする"
 	@echo " $$ make format"
+	@echo "フォーマットチェックをする"
+	@echo " $$ make format-check"
 	@echo "スタイルチェックを行う"
 	@echo " $$ make check_style"
+	@echo "デバッグフォルダを削除する"
+	@echo " $$ make clean"
+
+test:
+	python -m unittest
+
```

キャリブレーション用のコマンドを追加してほしいです。
（これがあってるかわからないけど）
```suggestion
calibrate:
        python -m src --calibrate
```